statistics/statistics.go

```diff
@@ -212,94 +276,135 @@ func (c *Column) mergeBuckets(bucketIdx int64) {
 	return
 }
 
-// columnToPB converts Column to ColumnPB.
-func columnToPB(col *Column) (*ColumnPB, error) {
-	data, err := codec.EncodeValue(nil, col.Values...)
+// Table represents statistics for a table.
+type Table struct {
+	Info    *model.TableInfo
+	Columns []*Column
+	Indices []*Column
```

Comment:
I have a different opinion. What you said is true, but they have more in common than difference, especially when you consider how they are used, for example, get row count in a range, once you encode the indices, the rest is the same. If we use a different type, will the same logic be writen twice? I do not know if this is worth the complexity.